homeassistant/package_constraints.txt

```diff
@@ -101,3 +101,7 @@ multidict>=6.0.2
 # Required for compatibility with point integration - ensure_active_token
 # https://github.com/home-assistant/core/pull/68176
 authlib<1.0
+
+# Required for compatibility with typer, used by pyunifiprotect integration
+# https://github.com/tiangolo/typer/pull/375
+click<=8.0.4"
```

Comment:
Is this correct? What does the `"` do after the version?

Reply:
You're right - it shouldn't be there!

I'm surprised it was accepted by PIP, and didn't get picked up by hassfest...